packages/muelu/src/Interface/MueLu_ML2MueLuParameterTranslator.cpp

```diff
@@ -490,6 +491,8 @@ namespace MueLu {
 
     mueluss << "</ParameterList>" << std::endl;
 
+    std::cout << mueluss.str() << std::endl;
```

Comment:
Is this related to @rstumin's questions?

Reply:
ah crud